public/app/plugins/panel/nodeGraph/NodeGraph.tsx

```diff
@@ -177,42 +189,46 @@ export function NodeGraph({ getLinks, dataFrames, nodeLimit }: Props) {
         </div>
       ) : null}
 
-      <svg
-        ref={panRef}
-        viewBox={`${-(width / 2)} ${-(height / 2)} ${width} ${height}`}
-        className={cx(styles.svg, isPanning && styles.svgPanning)}
-      >
-        <g
-          className={styles.mainGroup}
-          style={{ transform: `scale(${scale}) translate(${Math.floor(position.x)}px, ${Math.floor(position.y)}px)` }}
+      {dataFrames.length ? (
```

Comment:
Testing based off of dataFrames since if it's empty, then there are no nodes but it seems like the opposite isn't true (there could be dataFrames, but no nodes rendered because of screen position).